lib/matplotlib/__init__.py

```diff
@@ -1488,5 +1490,3 @@ def inner(ax, *args, data=None, **kwargs):
 
 # workaround: we must defer colormaps import to after loading rcParams, because
 # colormap creation depends on rcParams
-from matplotlib.cm import _colormaps as colormaps
-from matplotlib.colors import _color_sequences as color_sequences
```

Comment:
As the comment above notes, these imports are placed here on purpose to avoid circular imports. This is what's causing the test failure.